terraform/grafana.tf

```diff
@@ -92,6 +106,7 @@ resource "aws_cloudwatch_log_group" "grafana" {
   name = "${var.resource_prefix}-grafana"
 }
 
+# Not used anymore - temp because tf cannot destroy sg while attached
```

Comment:
tf gets stuck on deployment if I remove this... seems like terraform has issues identifying which security groups are still attached and it tries to destroy them (which is not allowed)... After deploying across all envs (dev, int, prod) we can remove this.